src/main/java/org/myrobotlab/service/InverseKinematics3D.java

```diff
@@ -239,9 +240,10 @@ public void publishTelemetry(String name) {
       // - 180 to + 180 ?
       double angle = MathUtils.radToDeg(theta) + l.getOffset();
       angleMap.put(jointName, (double) angle % 360.0F);
+      invoke("publishJointAngles", new AngleData(jointName, (double) angle % 360.0F));
       log.info("Servo : {}  Angle : {}", jointName, angleMap.get(jointName));
     }
-    invoke("publishJointAngles", angleMap);
+    // invoke("publishJointAngles", angleMap);

```

Comment:
This was useful to ensure that you have a consistent set of angles from which to do your updates.. this means that for the inmoov arm, you will need to publish 4x the number of messages to get the data.. and it's up to the listener to accumulate all of those together.. seems bad to remove this.